lib/util/env.js

```diff
@@ -0,0 +1,24 @@
+function getChildProcessEnv(customEnvVars = []) {
+  if (global.trustLevel === 'high') {
+    return process.env;
```

Comment:
To be safe, let's return a copy of process.env so that it doesn't matter if it's later modified.